docs/en/trusted-solutions/tier1/setup/s3_bucket.md

```diff
@@ -28,107 +30,45 @@ It's possible to start from the [Private S3 bucket AWS CloudFormation template](
   * Accept `I acknowledge that AWS CloudFormation might create IAM resources`
   * Next
 
-## Validation
+### CORS Configuration
+
+CORS policy on your S3 bucket will be minimally configured, allowing only the required origin (your site) and methods, amongst other things.
+
+### IAM Role
+
+The template will create an AWS Role with the following permissions on your bucket:
+
+* Allow Shotgun to access your S3 bucket.
+* Allow access to KMS (if you choose to activate AWS-KMS S3 encryption)
```

Comment:
I would not mention KMS, this could possibly add lot of complexity and not needed 99%